build_tools/azure/py38_conda_defaults_openblas_environment.yml

```diff
@@ -0,0 +1,20 @@
+channels:
+  - defaults
+dependencies:
+  - python=3.8
+  - numpy=1.17.3  # min
+  - blas[build=openblas]
+  - scipy=1.3.2  # min
+  - cython
+  - joblib
+  - threadpoolctl=2.2.0
+  - matplotlib=3.1.2  # min
+  - pandas
+  - pyamg
+  - pytest=6.2.5
```

Comment:
Ideally pinned versions should have an inline comment with the URL of an issue on our tracker that documents the pinning. In this case:

```suggestion
  - pytest=6.2.5  # https://github.com/scikit-learn/scikit-learn/issues/22396
```

Reply:
I haved added this kind of comments in the script that generates environments and lock files: https://github.com/scikit-learn/scikit-learn/pull/22448/files#diff-f09b752452c3a65895b05cf462a30cdbda554e33cb2003ddfe6c2e5cb8d2eca9R119

In my mind, the script has the overview of what is happening in all the builds and you don't necessarily want to look at individual environment files ...

Having said that, this is probably doable to add the kind of comment you want.